### ISSUE TITLE
Offer option to automatically replace/skip on file conflict if files are identical

### ISSUE THREAD

**Issue by mxmlnkn** [mate-desktop/caja#920]
It happens quite often, that I download some file twice in different folders and later try to merge the folders, it would be very convenient if instead of asking and manually comparing the shown file sizes, if caja would offer an option to just skip or replace all files if they are found to be identical.

**Comment by alexarnaud** [comment]
Right, I also expect to see this feature landed in Caja. I use rsync to figure out this issue but it's not user-friendly so.

Best regards.
